test/e2e/specs/liquidation.spec.js

```diff
@@ -124,5 +129,63 @@ describe('Wallet App Test Cases', () => {
         /Please increase your collateral or repay your outstanding IST debt./,
       );
     });
+
+    it('should wait and verify vaults are being liquidated', () => {
+      cy.contains(/3 vaults are liquidating./, {
+        timeout: LIQUIDATING_TIMEOUT,
+      });
+    });
+
+    it('should verify the value of startPrice from the CLI successfully', () => {
+      const fieldName = 'book0.startPrice';
+      const expectedValue = '9.99 IST/ATOM';
+
+      cy.checkFieldValue(fieldName, expectedValue).then(isMatched => {
+        expect(isMatched).to.be.true;
+      });
+    });
+
+    it('should verify the value of startProceedsGoal from the CLI successfully', () => {
+      const fieldName = 'book0.startProceedsGoal';
+      const expectedValue = '309.54 IST';
+
+      cy.checkFieldValue(fieldName, expectedValue).then(isMatched => {
+        expect(isMatched).to.be.true;
+      });
+    });
+
+    it('should verify the value of startCollateral from the CLI successfully', () => {
+      const fieldName = 'book0.startCollateral';
+      const expectedValue = '45 ATOM';
+
+      cy.checkFieldValue(fieldName, expectedValue).then(isMatched => {
+        expect(isMatched).to.be.true;
+      });
+    });
+
+    it('should verify the value of collateralAvailable from the CLI successfully', () => {
+      const fieldName = 'book0.collateralAvailable';
+      const expectedValue = '45 ATOM';
+
+      cy.checkFieldValue(fieldName, expectedValue).then(isMatched => {
+        expect(isMatched).to.be.true;
+      });
+    });
+
+    it.skip('should wait and verify vaults are liquidated', () => {
```

Comment:
please justify/explain the skipping in a comment

Reply:
The tests were running smoothly on my computer, but they failed in CI. I tried to replicate the CI environment on my local machine by updating a3p to latest version. Strangely, after updating, the tests failed on my local machine as well. I discovered that the tests pass when I use an older version of the container, specifically this one: `ghcr.io/agoric/agoric-3-proposals@sha256:c7cb71fb71227a52d407a2df7366571635c09a4acb4a30dc19a322ea9f9c6526`.